lib/jekyll/entry_filter.rb

```diff
@@ -52,7 +52,11 @@ def excluded?(entry)
     end
 
     def symlink?(entry)
```

Comment:
This should be a singleton method.  If you are initializing a class for a single method, it deserves to be a singleton.